date.go

```diff
@@ -182,6 +182,21 @@ func (d Date) YearDay() int {
 	return d.Time().YearDay()
 }
 
+// LeapYear gives whether or not the year is a leap year.
+// Any year divisible by 4 except century years
+// Any century year divisible by 400
+func (d Date) LeapYear() bool {
```

Comment:
Might be just as easy to use time.YearDay?

Reply:
Makes the below function redundant then too

Reply:
That's just the current day of the year though, right?

Reply:
If you create a time object on 1/1/XX and call YearDay it would give you the days in the year. Either 365 or 366, pretty neat 💪

Reply:
yeah, that's a different way of doing it